api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -107,58 +111,80 @@ public void validate(Object target, Errors errors) {
 				validateStopDatetime(visit, otherVisit, errors);
 			}
 		}
+		Date startDate = visit.getStartDatetime();
+		Patient patient = visit.getPatient();
+		Date birthDate;
+		if(patient!=null) {
+			birthDate = patient.getBirthdate();
+			if(startDate!=null && birthDate!=null) {
+				//Comparing patient's birthDate with their startDate to make sure that DOB is before startDate
```

Comment:
This comment is unnecessary. You only need to comment things that are not obvious to someone glancing at the code. (And in this case the next two lines tell the story "if ... compare ... error start date before birthdate".)